src/main.cpp

```diff
@@ -125,6 +125,18 @@ void handleInput (unsigned char key, int x, int y) {
       break;
     case 'h':
       displayUsage();
+      break;
+    case '1':
+      cursor->setShape(1);
```

Comment:
please add these commands to the help list (either under "other" or change "move cursor" to "cursor commands" and put them there